vnext/PropertySheets/Autolink.targets

```diff
@@ -4,6 +4,15 @@
  Licensed under the MIT License.. 
 -->
 <Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <!--
```

Comment:
This is taken from the patch targets. Context on why we need to move this here is in #5168